samples/subsys/logging/syst/sample.yaml

```diff
@@ -5,7 +5,6 @@ common:
   modules:
     - mipi-sys-t
   arch_exclude: mips nios2 posix sparc
```

Comment:
Some of the architectures listed in the "arch_exclude" list were due to lack of big endian support. I observe that an earlier commit removed the build-time check for big-endianness. From that, I take it that big endian support is now supported. If that is correct, then we should be able to remove some of the items from this list.

Reply:
The MIPI library may not work with other architectures due to va_list differences between them. So they need to be tested and enabled one by one.